site2/docs/io-elasticsearch-sink.md

```diff
@@ -6,6 +6,12 @@ sidebar_label: Elasticsearch sink connector
 
 The Elasticsearch sink connector pulls messages from Pulsar topics and persists the messages to indexes.
 
+## Requirements
+
+To deploy an ElasticSearch sink connector, the following are required:
```

Comment:
```suggestion
To deploy an Elasticsearch sink connector, meet the following requirements.
```

Reply:
The original seems better.

Reply:
1. Elasticsearch, lower case for "s" is correct
2. The subject of "To deploy an Elasticsearch sink connector" is "you" (user), so the subject of the latter sentence should be "you" as well. The subject of "meet the following requirements." is you.

Reply:
> 1. Elasticsearch, lower case for "s" is correct

Sure

> 2. The subject of "To deploy an Elasticsearch sink connector" is "you" (user), so the subject of the latter sentence should be "you" as well. The subject of "meet the following requirements." is you.

I'm a little not agree.  I prefer the original.